references/cuegen/generator.go

```diff
@@ -55,20 +48,24 @@ func NewGenerator(f string) (*Generator, error) {
 	types := getTypeInfo(pkg)
 
 	g := &Generator{
-		pkg:      pkg,
-		types:    types,
-		anyTypes: make(map[string]struct{}),
+		pkg:   pkg,
+		types: types,
```

Comment:
Maybe it's better to initialize opt in the `New func`.

Reply:
Updated